src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -35,10 +35,10 @@ angular
  * There is an example below of how this should look.
  *
  * ### Notes
- * The `md-autocomplete` uses the the [VirtualRepeat](api/directive/mdVirtualRepeatContainer)
+ * The `md-autocomplete` uses the the <a ng-href="api/directive/mdVirtualRepeatContainer">VirtualRepeat</a>
```

Comment:
Couldn't you use the `{@link}` directive here? AFAIK it reverses the URL properly, e.g.

```
{@link material.components.virtualRepeat}
```